api/src/main/java/org/apache/cloudstack/api/command/user/offering/ListDiskOfferingsCmd.java

```diff
@@ -42,6 +43,13 @@
     @Parameter(name = ApiConstants.NAME, type = CommandType.STRING, description = "name of the disk offering")
     private String diskOfferingName;
 
+    @Parameter(name = ApiConstants.ZONE_ID,
+            type = CommandType.UUID,
+            entityType = ZoneResponse.class,
+            description = "id of zone disk offering is associated with",
+            since = "4.13")
+    private Long zoneId;
```

Comment:
only for a zone, not for domains or multiple zones?

Reply:
@DaanHoogland List APIs are changed only for a single zone. Multiple domains and zones can be specified with create or update APIs.
Single zone in list API is needed as when we create storage, network or deploy VM we select or specify a particular zone. Therefore we need to query those offerings which are available for that particular zone or domain(through calling account).